front/src/modules/people/components/PeopleCard.tsx

```diff
@@ -0,0 +1,69 @@
+import { useNavigate } from 'react-router-dom';
+import styled from '@emotion/styled';
+
+import { Avatar } from '@/users/components/Avatar';
+import { Person } from '~/generated/graphql';
+
+export type PeopleCardPropsType = {
+  person: Pick<Person, 'id' | 'avatarUrl' | 'displayName' | 'jobTitle'>;
+};
+
+const StyledCard = styled.div`
+  align-items: center;
+  align-self: stretch;
```

Comment:
idem!

Reply:
We are actually using this one. Using `width` would result in something like below:

<img width="360" alt="Screenshot 2023-08-09 at 23 45 35" src="https://github.com/twentyhq/twenty/assets/140154534/0bfd8df2-4880-4005-b118-c0f5c38b9873">